python/measurement/average_taxons.py

```diff
@@ -0,0 +1,16 @@
+import gzip
+from lib import json_arrays
+from lib.helpers import dig
+from lib import services
+from statistics import mean
+
+
+def measure_average_taxons(filename):
+
+    def __number_of_taxons(content_item):
+        return len(dig(content_item, 'links', 'taxons') or [])
```

Comment:
Are these going to be just taxons in the Topic Taxonomy, or all taxons?

Reply:
That's a good point. The ticket is a bit vague on this point - At the moment it will calculate
Sum(all in scope content with any taxon) / Sum(all in scope content).

If we discount the taxons that are not in the Topic Taxonomy, should we then not also adjust the Divisor in some way?

Reply:
Unfortunately, the ticket is a bit old, and not very well defined. The best thing I can think to count is the average number of Topic Taxonomy taxons that content tagged to the Topic Taxonomy has. So, ignore in scope but untagged content as we are already measuring the proportion of tagged content, so I don't think this will add anything. Also ignore taxons outside of the Topic Taxonomy, as those could be confusing.

Reply:
fixed